autofaiss/external/quantize.py

```diff
@@ -100,15 +108,15 @@ def build_index(
     print(f"Using {nb_cores} omp threads (processes), consider increasing --nb_cores if you have more")
     faiss.omp_set_num_threads(nb_cores)
 
-    if isinstance(embeddings_path, np.ndarray):
+    if isinstance(embeddings_or_path, np.ndarray):
         tmp_dir_embeddings = tempfile.TemporaryDirectory()
-        np.save(tmp_dir_embeddings.name + "/emb.npy", embeddings_path)
-        embeddings_path = tmp_dir_embeddings.name
+        np.save(os.path.join(tmp_dir_embeddings.name, "emb.npy"), embeddings_or_path)
+        embeddings_or_path = tmp_dir_embeddings.name
```

Comment:
Should be embeddings_path starting from this point in the code: it becomes only a path in both branches

Reply:
Done